src/components/rating/Star.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Icon, {types, colors} from '../icons/Icon';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+const Star = ({size, checked, onClick = () => undefined}) => {
```

Comment:
I can create kind of helper with return empty function and we will use just reference to this.